extensions/CheckUser/api/ApiQueryCheckUser.php

```diff
@@ -113,7 +113,7 @@ public function execute() {
 						'timestamp' => wfTimestamp( TS_ISO_8601, $row->cuc_timestamp ),
 						'ns'        => intval( $row->cuc_namespace ),
 						'title'     => $row->cuc_title,
-						'user'      => User::getUsername( $row->cuc_user, $row->cuc_user_text ), // SUS-2944
+						'user'      => User::getUsername( $row->cuc_user, $row->cuc_ip ), // SUS-2944 / SUS-3080
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app